packages/dd-trace/test/debugger/devtools_client/breakpoints.spec.js

```diff
@@ -376,7 +379,7 @@ describe('breakpoints', function () {
 
         await breakpoints.removeBreakpoint({ id: 'probe-1' })
 
-        expect(sessionMock.post.callCount).to.equal(0)
+        expect(sessionMock.post).to.not.have.been.called
```

Comment:
I am not sure if a proxy is used here, but if not, having a typo would make the test pass. I therefore always prefer a function to be called at the end. That way it's guaranteed to be detected in case there is a typo.

Reply:
If you misspell it, it does error out:
<img width="739" alt="image" src="https://github.com/user-attachments/assets/97eeba57-1cd0-4d24-ba3e-e9ed0f3ff9ab" />

Reply:
The reason why I made these changes in this PR is that it provides better error messages in case of an assertion error and is less verbose